### PR TITLE
Replace tokens in remaining components

### PR DESCRIPTION
**Pull request by chukarave**
This replaces the last of the Wikit tokens found in components. 
To be merged only after #800 

Bug: T352311

### CODE REVIEW

File: resources/sass/app.scss [hunk 1]
```diff
@@ -14,10 +14,16 @@ html {
 
 body {
     @include body-M;
-    @include body-S;
     height: 100%;
 }
 
+// TODO: This is a temporary fix until issue https://phabricator.wikimedia.org/T326900
+// is marked as resolved.
+.cdx-button .cdx-icon, .cdx-button {
```
Comment (chukarave) [inline resources/sass/app.scss:22]: this is the way we found to make the mixin apply for buttons with and without Icons. This also adds the bold font weight, we might consider just adding it to the mixin itself?

Comment (sai-san) [reply resources/sass/app.scss:22]: I would encourage us to add the weight as an argument in the mixin, yes.

**Comment by sai-san** [comment]
UI review done. All replacements seem correct 👍

**Review by guergana — CHANGES_REQUESTED** [review]
This one is not compiling locally for me. Same error as in the failed browser test:

```
ERROR in ./resources/sass/app.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
Undefined mixin.
   ╷
23 │     @include ui-text-M;
```

it seems that this PR needs to be rebased including the first PR in the chain where this token was updated

File: resources/sass/app.scss (hunk 1 above)
Comment (guergana) [reply resources/sass/app.scss]: The `.cdx-button .cdx-icon` part is causing the icon to be rendered in a different color, any properties assigned to the parent `.cdx-button` will be overwritten here. I suggest getting rid of the `.cdx-button .cdx-icon` part here.

**Review by guergana — CHANGES_REQUESTED** [review]
spaces, nitpicks that will give linting problems later.

File: resources/sass/_typography.scss [hunk 2]
```diff
@@ -25,14 +24,15 @@
...
+   color: $color-base;
 }
 
 @mixin ui-text-M ($font-weight: $font-weight-normal){
     font-family: $font-family-system-sans;
     font-size: $font-size-medium;
     font-weight: $font-weight;
     line-height: $line-height-xx-small;
-    color: $color-base;
+   color: $color-base;
```
Comment (guergana) [inline resources/sass/_typography.scss:35]: this is misaligned :grimacing:

**Review by guergana — CHANGES_REQUESTED** [review]

File: resources/sass/_typography.scss (hunk 2 above)
Comment (chukarave) [reply resources/sass/_typography.scss:35]: In more than one place even

**Review by guergana — CHANGES_REQUESTED** [review]
Hi, i can't write on that line but there's also an extra double new line in lines 412-413 https://github.com/wmde/wikidata-mismatch-finder/pull/823/files#diff-b339140a5abe2cafdc4e45227556c9b8799da604d97644e62f472d5f80506725R412

**Comment by guergana** [comment]
I guess more of a question for UX @sai-san : if the log in link in the AuthWidget is not supposed to have a visited state (aka purple color after visited)... is it supposed to have a visited state when it is part of the message component? 
image

**Comment by chukarave** [comment]
@guergana this is a good catch :+1: the link styles are part of [T347161](https://phabricator.wikimedia.org/T347161), please add this comment there so it will not get lost

**Pull request merged**